recipes-kernel/gpio-firecrest-driver/gpio-firecrest-driver_0.1.bb

```diff
@@ -2,7 +2,7 @@ SUMMARY = "Firecrest driver"
 DESCRIPTION = "${SUMMARY}"
 LICENSE = "GPL-2.0-only"
 LIC_FILES_CHKSUM = "file://COPYING;md5=12f884d2ae1ff87c09e5b7ccc2c4ca7e"
-DRIVERBRANCH = "gpio_firecrest"
+DRIVERBRANCH = "main"
 SRCREV = "${AUTOREV}"
```

Comment:
we should not use ${AUTOREV} when released. the reason is that we should be able to backtrack customer builds based on our bsp layer.